gateway/src/constructors/express/routes/key/index.js

```diff
@@ -0,0 +1,23 @@
+const axios = require('axios')
+const router = require('express').Router()
+
+
+module.exports = (context) => {
+ const { sequelize, models } = context;
```

Comment:
If we prefer, these may be pulled straight from the context (since we know they're always there):

```
module.exports = ({ sequelize, models, ...context }) => {
```